### PR TITLE
Added account number field

### PR DESCRIPTION
**Pull request by aonomike**
Account number field added to the UI

### CODE REVIEW

File: src/main/java/org/drip/model/DripUser.java [hunk 1]
```diff
@@ -16,6 +16,8 @@
 	private String lastName;
 	@Column(name="business_name")
 	private String businessName;
+	@Column(name="account_number")
+	private int accountNumber;
```
Comment (geoff-wasilwa) [inline src/main/java/org/drip/model/DripUser.java]: @aonomike change account number to string

File: src/main/java/org/drip/model/User.java [hunk 2]
```diff
@@ -20,6 +20,10 @@
 	@Id
 	@GeneratedValue(strategy=GenerationType.IDENTITY)
 	private long id;
+	
+	@Column(name="account_number")
+	private int accountNumber;
```
Comment (geoff-wasilwa) [inline src/main/java/org/drip/model/User.java]: @aonomike remove account number from this class

File: src/main/java/org/drip/model/DripUser.java (hunk 1 above)
Comment (geoff-wasilwa) [inline src/main/java/org/drip/model/DripUser.java]: Given that one user can have several accounts, maybe we should move this elsewhere. Maybe have an `AccountNumber` class with `id`

Comment (aonomike) [inline src/main/java/org/drip/model/DripUser.java]: Okay. On it

Comment (aonomike) [inline src/main/java/org/drip/model/DripUser.java]: @geoff-wasilwa  and the account number still remains a String?

**Comment by geoff-wasilwa** [comment]
@aonomike change `DripUser.findByFirstNameAndLastNameAndAreaCodeAndPhoneNumberAndZipCode` to something like:
```java
@Query("SELECT user FROM User left join fetch user.accountNumbers WHERE user.firstName =:firstName and user.lastName =:lastName and user.phoneNumber =:phoneNumber and user.zipCode =:zipCode and user.areaCode =:areaCode")
public User findUserBy(@Param("firstName")  String firstName, @Param("lastName") String lastName, @Param("phoneNumber") String phoneNumber, @Param("zipCode") String zipCode, @Param("areaCode") String areaCode);
```

**Pull request closed without merging**